media-gfx/qview/qview-3.0.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Minimalist image viewer"
+DESCRIPTION="qView is an image viewer designed with minimalism and usability in mind."
+HOMEPAGE="https://interversehq.com/qview/"
+COPYRIGHT="2018-2020 jurplel and qView contributors"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/jurplel/qView/archive/3.0.zip"
```

Comment:
Where possible we try to use $portVersion (keeps future updates on github clean)
```diff
- SOURCE_URI="https://github.com/jurplel/qView/archive/3.0.zip"
+ SOURCE_URI="https://github.com/jurplel/qView/archive/$portVersion.zip"
```